### DISCUSSION TITLE
How is route determined in EDScout?

### DISCUSSION THREAD

**Discussion by Tailspinner** [joncage/ed-scout#162] [General]
How is route determined in EDScout? How far off route will it look, and can this be adjusted? It is very useful!

**Comment by joncage** [comment]
It literally just looks up those systems that the galaxy map finds. It doesn't look outside that route at all as it's just reading from the games logs.

There was a suggestion to find off-route systems but it's a non-trivial exercise: https://github.com/joncage/ed-scout/issues/92
